style.css

```diff
@@ -14,65 +10,78 @@ body {
 	
 }
 
+p {
+	font-family: "Source Sans Pro", sans-serif;
+	font-size: 1.5rem;
+}
+
+h1, h2, h3, h4, h5, h6 {
+	font-family: 'Poppins', sans-serif;
+}
+
+h2 {
+	font-size: 3rem;
+}
+
+h3 {
+	font-size: 2rem;
+}
+
 .navigationBar {
 	display: grid ;
 	grid-template-columns: 25% 75%;
 	background-color: #270483;
-/*	background-color: #630480;*/
-	height: 60px;
-	position: fixed; /* Set the navbar to fixed position */
-	top: 0; /* Position the navbar at the top of the page */
-	width: 100%; /* Full width */
+	position: fixed;
+	top: 0;
+	width: 100%;
 	z-index: 10;
+}
 
+.welcome {
+	padding-top: 6vh;
 }
 
 nav {
-	/*justify-self: end;*/
 	display: flex;
-	justify-content: space-evenly; 
-	
+	justify-content: space-evenly; 	
 }
 
 nav a {
 	align-self: center;
 	color: white;
 	font-family: 'Poppins', sans-serif;
 	text-decoration: none;
-	font-size: 2em;
-	padding: 8px;
+	font-size: 1.5rem;
+	padding: 1vh;
+	margin: 0.5vw;
```

Comment:
It doesn't really make sense to use `vh` and `vw` in this way because the padding and margin are being applied on all sides, whereas the units of measurement are dependent on horizontal size and vertical size of the window respectively. The size of padding and margin aren't really relevant to the size of the window anyway - you can resize the window to any size and that shouldn't really change the padding and margin on these links. Switch this out for pixels or rems.

If you don't understand this comment ask me and I will explain out loud!